dashboard/test/models/regional_partner_test.rb

```diff
@@ -65,4 +65,59 @@ class RegionalPartnerTest < ActiveSupport::TestCase
       regional_partner.program_manager = program_manager.id
     end
   end
+
+  test 'find_by_region finds matching regional_partner' do
+    # partners and their mappings
+
+    # regional_partner_WA
+    # regional_partner_WA_98105
+    # regional_partner_98105
+    # regional_partner_NY
```

Comment:
What are these comments?

Reply:
Sorry for not explaining - those comments were placeholders for test data scenarios that were not complete yet.  Most of the test scenarios are complete now.